habitat/core/simulator.py

```diff
@@ -105,10 +105,15 @@ def __init__(
 
 class RGBSensor(Sensor):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
+        self.uuid = "rgb"
         super().__init__(*args, **kwargs)
+        if "config" in kwargs:
+            config = kwargs["config"]
+            if hasattr(config, "UUID"):
+                self.uuid = config.UUID
 
     def _get_uuid(self, *args: Any, **kwargs: Any) -> str:
-        return "rgb"
+        return self.uuid
```

Comment:
This all can be done with:
```suggestion
        return self.uuid
         if hasattr(self.config, "UUID"):
           return config.UUID
         else:
           return "rgb"
```